Dockerfile

```diff
@@ -27,13 +30,25 @@ RUN microdnf update -y && \
         python3.11-pip \
         python3-pip \
         python3-dnf \
-        nodejs \
-        npm \
-        skopeo && \
+        python3.9 \
+        cargo \
+        golang \
+        podman \
```

Comment:
```suggestion
```

I think we can drop podman, it was required before skopeo became the go-to thing.